.github/workflows/_run.yml

```diff
@@ -234,6 +234,16 @@ jobs:
 
     - run: |
         mkdir ${{ runner.temp }}/container
+        if mountpoint -q /mnt; then
+            echo "has-mnt=true" >> "$GITHUB_OUTPUT"
+        fi
+        USAGE="$(df --output=pcent /mnt | tail -n 1 | tr -d ' %')"
```

Comment:
If `/mnt` is not a mountpoint, df will return the usage of `/`. Is that what we want? Or maybe put everything within the `if` above?

Reply:
yeah - lets move it into the if clause - altho tbh if `/` is 100% full we have even bigger problems

Reply:
yeah, ok - i was kinda ambivalent about whether to trigger all the time - but yeah makes sense